R/qc_sequence_coverage.R

```diff
@@ -77,7 +77,7 @@ qc_sequence_coverage <- function(data,
       size = 1
     ) +
     ggplot2::geom_vline(data = result %>% dplyr::distinct(.data$median_coverage, {{ sample }}),
-                        mapping = aes(xintercept = median_coverage),
+                        mapping = aes(xintercept = .data$median_coverage),
```

Comment:
Shouldn't we remove the `.data`- since it was deprecated in tidyselect 1.2.0? 
https://community.rstudio.com/t/use-of-data-in-tidyselect-expressions-is-now-deprecated/150092
`Use of .data in tidyselect expressions is now deprecated to more cleanly separate tidy-select from data-masking. Replace .data$x with "x" and .data[[var]] with any_of(var) or all_of(var) (#169).`

Reply:
This only affects "tidyselect expressions" and not the masking expressions. So as far as I am aware it does not affect the mapping of ggplot.